bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -339,7 +339,7 @@ <h2>{{ ftl('home-that-one-time') }}</h2>
     </div>
   </section>
 
-  <aside class="c-newsletter">
+  <aside class="c-newsletter" aria-label="{{ ftl('newsletter-form-label') }}">
     <div class="c-newsletter-wrapper mzp-l-content mzp-t-content-lg">
       <div class="c-newsletter-info">
```

Comment:
Would it make sense to also use this newly added string as a `legend` for `fieldset.mzp-c-newsletter-content` there, that's missing one?